src/boot/bootctl.c

```diff
@@ -52,6 +53,11 @@ static bool arg_print_dollar_boot_path = false;
 static bool arg_touch_variables = true;
 static PagerFlags arg_pager_flags = 0;
 static bool arg_graceful = false;
+/* Because JSON output only supports 'short' and 'pretty', this could be a bool,
+ * but, I do an int here so that other formats can be added in the future if the
+ * need arrises.
+ */
```

Comment:
please drop the comment. it's unnecessary and actually wrong. It couldn't be a bool, since "off", "short" and "pretty" are three possible values, not three.